sorc/build_gsi_monitor.sh

```diff
@@ -1,80 +1,40 @@
 #! /usr/bin/env bash
 set -eux
 
-source ./machine-setup.sh > /dev/null 2>&1
```

Comment:
I think this may need to stay to support partial build? If not, we should take it out of all the build scripts.

Reply:
Can you verify it is needed for partial build?
I tried with a partial build for the GSI, GSI-utils and GSI-monitor and had no impact.
I think it is necessary for builds that loads modules in the build scripts in this repo.
It is not necessary for GSI, GSI-utils or GSI-monitor that uses the `build.sh` from their repos.

Reply:
Got it